docs/docs/rad-lab-ui/ui_installation/env-variables.md

```diff
@@ -35,6 +35,7 @@ Firebase environment variables can be found in project settings section of your
 - `NEXT_PUBLIC_NOTIFICATION_TOPIC` - Topic created for notifications
 - `NEXT_PUBLIC_NOTIFICATION_SUB` - Subscription created for notifications
 - `NEXT_PUBLIC_GIT_API_URL` - Url of the public repo to pull the deployment modules
+- `SECRET_MANAGER_LOCATION` - Google Cloud region of the Secret Manager instance
```

Comment:
2 Questions:

1. **How are we setting this Environment variable ?** Are we needing admins to manually update the .env file with the region before UI deployment  or using the existing [bash script](https://github.com/GoogleCloudPlatform/rad-lab/blob/main/radlab-ui/webapp/gen_local_config.sh) which creates the .env file.

2. **What is the default value of this variable ?** Is it the same as [app_engine_location variable](https://github.com/GoogleCloudPlatform/rad-lab/blob/31f7260e19cd185614ba737dd553fa51ba8d2c69/radlab-ui/automation/terraform/infrastructure/variables.tf#L23C1-L27C2) ? 

**SOLUTION:** If its manually set and we want to automate it like other variables we can expose the above `app_engine_location` variable as a [TF output](https://github.com/GoogleCloudPlatform/rad-lab/blob/main/radlab-ui/automation/terraform/infrastructure/outputs.tf) and we can use the existing [bash script](https://github.com/GoogleCloudPlatform/rad-lab/blob/main/radlab-ui/webapp/gen_local_config.sh) to create the .env with the same.

Reply:
`app_engine_location` and `Secret Manager Location` are different things so moved the env to `outputs`.